website/docs/more/deployment.mdx

```diff
@@ -0,0 +1,43 @@
+---
+title: "Deployment"
+description: "Deploying Yew applications"
+---
+
+When you are ready to deploy your Yew application to a server, you have various options for deployment.
+
+`trunk build --release` builds your app in release mode. Set up your HTTP server so that to serve `index.html` whenever your site is visited, and requests to static paths like `index_<hash>.js` and `index_bg_<hash>.wasm` are served with the contents of their respective contents from the dist directory generated by trunk.
```

Comment:
```suggestion
`trunk build --release` builds your app in release mode. Set up your HTTP server so that it serves `index.html` whenever your site is visited, and requests to static paths like `index_<hash>.js` and `index_bg_<hash>.wasm` are served with the contents of their respective contents from the dist directory generated by trunk.
```